app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -414,37 +384,33 @@ class GithubWebhook extends RequestHandler<Body> {
     }
   }
 
-  Future<void> _pingForTriage(
-      GitHub gitHubClient, PullRequestEvent event) async {
+  Future<void> _pingForTriage(GitHub gitHubClient, PullRequest event) async {
     final String body = config.goldenTriageMessage;
-    final RepositorySlug slug = event.repository.slug();
+    final RepositorySlug slug = event.head.repo.slug();
     await gitHubClient.issues.createComment(slug, event.number, body);
   }
 
   Future<void> _checkForLabelsAndTests(
-    PullRequestEvent event,
-    bool isDraft,
+    String eventAction,
+    PullRequest event,
   ) async {
-    if (event.repository.fullName.toLowerCase() == 'flutter/flutter') {
+    if (needsCQLabelList.contains(event.head.repo.fullName.toLowerCase())) {
       final GitHub gitHubClient = await config.createGitHubClient();
       try {
```

Comment:
needsCQLabelList should only be used to require CQ+1 for presubmits and not as a synonym of 'flutter/flutter'.